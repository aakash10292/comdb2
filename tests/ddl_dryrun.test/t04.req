dryrun alter table t1 add column a int not null

dryrun alter table t1 add column b byte(-5) 

dryrun alter table t1 drop column id 

dryrun alter table t2 alter column f set data type byte(7)

dryrun alter table t3 add index KEY_12345678901234567890123456789012345678901234567890(a,b) 

drop table t3
