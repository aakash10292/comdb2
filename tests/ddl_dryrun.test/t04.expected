[dryrun alter table t1 add column a int not null] failed with rc 240 Cannot add new field without dbstore or null
[dryrun alter table t1 add column b byte(-5)] failed with rc -3 Invalid type specified.
[dryrun alter table t1 drop column id] failed with rc 240 constraint error for table "t2" key "k" -> <"t1":"key">: parent key not found
[dryrun alter table t2 alter column f set data type byte(7)] failed with rc 240 Cannot change size of byte array without dbpad

[dryrun alter table t3 add index KEY_12345678901234567890123456789012345678901234567890(a,b)] failed with rc 240 Failed to process schema!

